server/controllers/party.js

```diff
@@ -103,7 +111,38 @@ static updateParty(req, res) {
       });
       return;
     }
-  
+  /**
+ * @function deleteParty
+ * @memberof PartyController
+ * @static
+ */
+static deleteParty(req,res){
+    let { id } = req.params;
+    if(isNaN(id)){
+        res.status(404).send({
+            status: 404,
+            message:'Enter the correct party parameter'
+        })
+        return;
+    } 
+    const partyId = Number(id);
+    const partyDetails = party.find(c => c.id === partyId);
+    if (!partyDetails) {
+      res.status(404).send({
+        status: 404,
+        message: 'The party with given id was not found',
+      });
+      return;
+    }
+    const filteredParty = party.filter(parties => parties !== partyDetails);
+    // const filteredParty = party.slice(0, partyId-1).concat(party.slice(partyId, party.length))
```

Comment:
Hi @akinyeleolat try and do away with commented out code when pushing

Reply:
Noted